components/DateSelector.vue

```diff
@@ -60,6 +52,10 @@
                 }
             },
 
+            disabled: {
+                type: Object
```

Comment:
is this parameter required? if not it should have a default
```default: () => new Object()```

Reply:
It doesn't have a default because that would trigger the actual check for disabled values. I bi-pass all calculations if it doesn't exist.

Reply:
I think you can default it as null then, null is technically an object